lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1573,6 +1573,15 @@ def _resource_sku_to_capability(  # noqa: C901
                 # will revert it till bug fixed.
                 if "standardDCSv2Family" == resource_sku.family:
```

Comment:
Because there are multiple items now, to reduce code lines, it should be like `if esource_sku.family in ["standardDCSv2Family", "standardNCSv2Family"]:`.

Reply:
fixed